pkg/monitortestframework/impl.go

```diff
@@ -115,7 +115,10 @@ func (r *monitorTestRegistry) StartCollection(ctx context.Context, adminRESTConf
 					},
 					SystemOut: fmt.Sprintf("failed during setup\n%v", err),
 				}
-				return
+				var flakeErr *NotSupportedError
+				if !errors.As(err, &flakeErr) {
+					return
+				}
```

Comment:
Is *NotSupportedError the right type? I'm confused by that usage here

Reply:
Sigh.